kubectl-cloudflow/config/config.go

```diff
@@ -361,34 +361,31 @@ func labelHasPrefix(label string) bool {
 	return strings.Count(label, "/") == 1 && !strings.HasPrefix(label, "/") && !strings.HasSuffix(label, "/")
 }
 
-func validateLabel(name string, prefix string) error {
-	// See https://github.com/kubernetes/kubernetes/issues/71140, IsDNS1123Subdomain and IsDNS1123Label do not allow uppercase letters.
-	labelPattern := regexp.MustCompile(`^[a-z0-9A-Z]{1}[a-z0-9\.\_\-]{0,61}[a-z0-9A-Z]{1}$`)
-
-	// TODO a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character
+//Only the prefix of a value key has a different check
```

Comment:
I've read that prefixes can't have uppercase, is that what you mean?

Reply:
Oh, I don't think so. I was reading the code and didn't remember that. I will add a test for that otherwise may happen later on. Good catch.